lxd/cgroup/abstraction.go

```diff
@@ -3,8 +3,10 @@ package cgroup
 import (
 	"fmt"
 	"strconv"
+	"strings"
 
 	"github.com/lxc/lxd/shared"
+	"github.com/lxc/lxd/shared/metrics"
```

Comment:
I think I'd prefer our `cgroup` package not to return data that's specific to metrics.
In this case, it looks like we could probably return a `map[string]uint64` and simplify the function quite a bit, then doing the needed reshuffling on the metrics side of things.